xtask/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "xtask"
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.83"
```

Comment:
ditto: inherit from workspace - ??

Reply:
See further discussion from the other thread here: https://github.com/gfx-rs/wgpu/pull/7036#discussion_r1936277983